zerver/lib/markdown/help_settings_links.py

```diff
@@ -79,6 +79,16 @@
         "Bots",
         "/#organization/bots",
     ],
+    "all-bots": [
+        "Organization settings",
+        "Bots",
+        "/#organization/bots/all-bots",
+    ],
+    "org-your-bots": [
```

Comment:
in that case it will get conflicted with `"your-bots": ["Personal settings", "Bots", "/#settings/your-bots"],` perhaps I have change both of them to `your-bots-via-personal-settings` and `your-bots-via-org`. kindly your feedback.